cmd/server/main.go

```diff
@@ -48,6 +49,10 @@ import (
 )
 
 var (
+	// custom GitHub API URLs
+	githubAPIRawURL = flag.String("github-api-base-url", "", "base URL for GitHub API.  Please set this when you use GitHub Enterprise. This often is your GitHub Enterprise hostname. If the base URL does not have the suffix \"/api/v3/\", it will be added automatically.")
+	githubAPIRawUploadURL = flag.String("github-api-upload-url", "", "upload URL for GitHub API. This must be set when you set --github-api-base-url. This often is your GitHub Enterprise hostname. If the upload URL does not have the suffix \"/api/uploads\", it will be added automatically.")
```

Comment:
Can this flag be removed?

- Triage Party should not perform uploads - it's read-only.
- As far as I know, the upload URL is usually the same as the base URL.

Reply:
Oh, that's good news.  I will remove it.

> As far as I know, the upload URL is usually the same as the base URL.

actually, it is slightly different.

- `baseURL`: `https://my.github.enterprise/api/v3/`
- `uploadURL`: `https://my.github.enterprise/api/uploads/`

But, it's not a problem anyway.